frontend/src/app/app.module.ts

```diff
@@ -130,10 +131,14 @@ const appRoutes: Routes = [
     component: PasswordResetComponent,
     canActivate: [LoggedOutGuard],
   },
+  {
+    path: 'me',
+    component: MeComponent,
+  },
```

Comment:
I think you can use guard here, to block access to this site for an anonymous user.